src/haproxy/Dockerfile

```diff
@@ -1,10 +1,3 @@
-FROM haproxy:1.9-alpine
-
-VOLUME [ "/certs" ]
-
-RUN apk add --update inotify-tools
+FROM balena/open-balena-haproxy:v4.0.6
```

Comment:
This is problematic because we're hiding the actual dependency inside a bespoke service. I believe the only reason for this is we need a way to pass configuration. Is there a better way?

There's #127 if you'd rather we tackled this separately.

Reply:
I am not aware of a "better" way, no.

Reply:
Do we not have a similar use case elsewhere? The VPN? How does it get its config?

Reply:
> This is great progress and a necessary step towards an easily deployable and testable openBalena but the fact it completely disregards support for any OS other than balenaOS makes it a no-go as it is. We really need to work on identifying what is balenaOS-specific and bridge the gap.

Should I close the PR then or will you work on adding other OS support @dfunckt ?